packages/office-ui-fabric-react/src/common/_semanticSlots.scss

```diff
@@ -81,15 +66,16 @@ $menuItemBackgroundHoveredColor: '[theme:menuItemBackgroundHovered, default: #f4
 $menuItemBackgroundPressedColor: '[theme:menuItemBackgroundPressed, default: #eaeaea]';
 $menuItemTextColor: '[theme:menuItemText, default: #333333]';
 $menuItemTextHoveredColor: '[theme:menuItemTextHovered, default: #212121]';
-
-/* Lists */
 $listBackgroundColor: '[theme:listBackground, default: #ffffff]';
 $listTextColor: '[theme:listText, default: #333333]';
-$listItemBackgroundHoveredColor: '[theme:listItemBackgroundHovered, default: #f8f8f8]';
+$listItemBackgroundHoveredColor: '[theme:listItemBackgroundHovered, default: #f4f4f4]';
 $listItemBackgroundCheckedColor: '[theme:listItemBackgroundChecked, default: #eaeaea]';
-$listItemBackgroundCheckedHoveredColor: '[theme:listItemBackgroundCheckedHovered, default: #d0d0d0]';
+$listItemBackgroundCheckedHoveredColor: '[theme:listItemBackgroundCheckedHovered, default: #dadada]';
 $listHeaderBackgroundHoveredColor: '[theme:listHeaderBackgroundHovered, default: #f4f4f4]';
 $listHeaderBackgroundPressedColor: '[theme:listHeaderBackgroundPressed, default: #eaeaea]';
-
-// DEPRECATED SLOTS
-$menuItemBackgroundCheckedColor: '[theme:menuItemBackgroundChecked, default: #eaeaea]';
+$actionLinkColor: '[theme:actionLink, default: #333333]';
+$actionLinkHoveredColor: '[theme:actionLinkHovered, default: #212121]';
+$linkColor: '[theme:link, default: #0078d4]';
+$linkHoveredColor: '[theme:linkHovered, default: #004578]';
+$listTextColorColor: '[theme:listTextColor, default: #333333 /* @deprecated */]';
```

Comment:
instead of nesting the `@deprecated` comment within the string, could you include that outside?